### ISSUE TITLE
Add API for pre-filling username

### ISSUE THREAD

**Issue by mvysny** [vaadin/vaadin-login-flow#77]
Sometimes it's good to pre-fill the username, e.g. if the username is a phone then it's good to add `+358`.

Workaround:

```
UI.getCurrent().getPage().executeJavaScript("document.getElementById(\"vaadinLoginUsername\").value = '+358';");
```

**Comment by sebdavid** [comment]
I agree with that. But even if it is a regular username/login/etc. , it may be helpful to initialize it in some cases.
The workaround does not work on my side, neither the following : 
```java
loginForm.getElement().executeJavaScript("this.querySelector('...').value = $0", username);
```
The result of the call to querySelector is null at the time of execution. I'm using Vaadin13.0.13. I don't know why it is null, maybe because of the page construction.
 But if I delay the call with setTimeout(..., 500) in the JS expression, it works. But this is definitely not a good solution.
